lib/content-types/load.js

```diff
@@ -3,28 +3,33 @@
 const dir = require('node-dir');
 const yaml = require('js-yaml');
 const slugify = require('lodash/kebabCase');
-const globalConfig = require('config');
 const path = require('path');
 const utils = require('../utils');
 
 
 /**
  * Gets all Content Types
  *
+ * @param {object} configuration - configuration object
+ *
  * @returns {Promise} - An {Array} containing {Object} for each content type, including `name` {string}, `id` name {string}, and full config {object} loaded from YAML config file
  */
-const load = () => {
+const load = (configuration) => {
```

Comment:
Can we keep the parameter name consistent throughout the project for configuration?